index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Terri's CSS Tutorials 1-5 Github Page</title>
+  </head>
+  <body>
+    <strong>This Github Page reflects:</strong>
+    <ul>
+      <li>
+        my work through the first five tutorials of Zastrow's How to Style HTML
+        with CSS.
+      </li>
+      <li>my application of that knowledge while flexing my Github Skills.</li>
+    </ul>
+
+    <strong>Here are links to my implementations of the tutorial concepts:</strong>
```

Comment:
These `strong` blocks make more sense as `h1` and then `h2` blocks semantically.

From W3:
> A semantic element clearly describes its meaning to both the browser and the developer.

i.e. If the content is visually and logically a heading, mark it with a `h#` tag.